relayer/relays/parachain/beefy-listener.go

```diff
@@ -674,6 +703,37 @@ func (li *BeefyListener) scanForCommitments(
 	return tasks, nil
 }
 
+func fetchBundleProof(
+	basicChannelBundleProof *MerkleProof,
+	api *gsrpc.SubstrateAPI,
+	digestItem AuxiliaryDigestItem,
+	bundleIndex int,
+) error {
+	var proofHex string
+	err := api.Client.Call(&proofHex, "basicOutboundChannel_getMerkleProof", digestItem.AsCommitment.Hash.Hex(), bundleIndex)
+	if err != nil {
+		return fmt.Errorf("call rpc: %w", err)
```

Comment:
Should also log the call params here:

```suggestion
		return fmt.Errorf("call rpc basicOutboundChannel_getMerkleProof(%v, %v): %w", digestItem.AsCommitment.Hash.Hex(), bundleIndex , err)
```